src/hooks/users/useWorldUsersById.ts

```diff
@@ -29,16 +27,3 @@ export const useWorldUsersById = () => {
     worldUsersById,
   };
 };
-
-// @debt typing, this uses Partial<Record<K,T>> to work around the bug where Record implies that a User will exist for literally any given string, which is untrue
-// @debt refactor all usages of Record<string, User> to Partial<Record<string, User>> (then we can remove this workaround)
-export const useWorldUsersByIdWorkaround = () => {
-  const {
-    worldUsersById: _worldUsersById,
-    isWorldUsersLoaded,
-  } = useWorldUsersById();
-
-  const worldUsersById: Partial<Record<string, User>> = _worldUsersById;
-
-  return { worldUsersById, isWorldUsersLoaded };
-};
```

Comment:
Given that we will remove this from the codebase, I decided to simplify this task and to leave only one `wrong` implementation